cli/add.go

```diff
@@ -216,15 +176,37 @@ func add(c *cli.Context) error {
 				log.Fatalf("[ENVMAN] Failed to load envman config: %s", err)
 			}
 
-			value, err = readMaxWithTimeout(os.Stdin, uint(configs.EnvBytesLimitInKB*1024), 1*time.Second)
+			data, err := ioutil.ReadAll(os.Stdin)
 			if err != nil {
-				if err == errTimeout {
-					log.Warning("[ENVMAN] Standard input read timed out")
-				} else {
-					log.Fatalf("[ENVMAN] Failed to read standard input: %s", err)
+				log.Fatalf("[ENVMAN] Failed to read standard input: %s", err)
+			}
+
+			if configs.EnvBytesLimitInKB > 0 && len(data) > configs.EnvBytesLimitInKB*1024 {
+				valueSizeInKB := ((float64)(len(data))) / 1024.0
+				log.Warnf("environment value (%s...) too large", value[0:100])
+				log.Warnf("environment value size (%#v KB) - max allowed size: %#v KB", valueSizeInKB, (float64)(configs.EnvBytesLimitInKB))
```

Comment:
why `%#v` ?

Reply:
%#v	a Go-syntax representation of the value: https://golang.org/pkg/fmt/
so it is fine